packages/OmegaPrint-Core.package/OPPrinter.class/instance/include.between.and..st

```diff
@@ -0,0 +1,6 @@
+helper
```

Comment:
Wir haben gerade sehr viele Kategorien mit nur einer Methode, das ist glaube ich noch nicht so schoen. Wie man hier sieht, wird nie `self` gerufen, eigentlich gehoert diese Methode wahrscheinlich in eine andere Klasse